### ISSUE TITLE
Notes disappear from the Notes Menu

### ISSUE THREAD

**Issue by HATEthePLOT** [nextcloud/notes#68]
I have experienced that the app does not show any notes anymore. They are present in the notes folder, however, I cannot access or edit them with the app. The mobile-app from f-droid shows connection issues. This is interesting insofar as I did not change anything with the server and it worked pretty much out of the box. I am using version 2.2.0 on Nextcloud 11.0.1

**Comment by nonplusnl** [comment]
I have exactly the same experience, both in web-interface (most notes invisible, except most recent one), and in Android-app (connection issues). Logfile shows some issues, several unspecified exceptions, and this: 
Error        PHP	in_array() expects parameter 2 to be array, null given at /var/www/npc/apps/notes/service/notesservice.php#182	
Error	PHP	Undefined offset: 1574184 at /var/www/npc/apps/notes/service/notesservice.php#182

**Comment by Happyfeet01** [comment]
i have the same issue. 


````
in_array() expects parameter 2 to be array, null given at /var/www/vhosts/domain.tld/nextcloud/apps/notes/service/notesservice.php#182
Undefined offset: 43456 at /var/www/vhosts/domain.tld/nextcloud/apps/notes/service/notesservice.php#182
````

**Comment by Henni** [comment]
should be fixed by #85 
If this error still occurs please reopen this issue.
